### ISSUE TITLE
Loop when looking for files not in DB (theme.mp3)?

### ISSUE THREAD

**Issue by BrutuZ** [croneter/PlexKodiConnect#737]
Whenever I browse my TV shows, their theme plays on the background due to local `theme.mp3` files exported from Plex. Frontend it's all fine and dandy, the only issue is when I have to check the log file, it gets spammed with the lines below, even long after the mentioned playback has finished:
```
19:52:20.176 T:9840   DEBUG: PLEX.playqueue: Comparing new Kodi playqueue [{u'title': u'', u'type': u'unknown', u'file': u'/media/<SOME_TV_SHOW>/theme.mp3', u'label': u'/media/<SOME_TV_SHOW>/theme.mp3'}] with our play queue []
19:52:20.176 T:9840   DEBUG: PLEX.playqueue: Detected new Kodi element at position 0: {u'title': u'', u'type': u'unknown', u'file': u'/media/<SOME_TV_SHOW>/theme.mp3', u'label': u'/media/<SOME_TV_SHOW>/theme.mp3'}
19:52:20.176 T:9840   DEBUG: PLEX.playlist_func: Starting research for Kodi id since we didnt get one: {u'title': u'', u'type': u'unknown', u'file': u'/media/<SOME_TV_SHOW>/theme.mp3', u'label': u'/media/<SOME_TV_SHOW>/theme.mp3'}
19:52:20.179 T:9840   DEBUG: PLEX.kodi_db.video: Did not find any file, abort
19:52:20.179 T:9840   DEBUG: PLEX.kodi_db: No kodi video db element found for path /media/<SOME_TV_SHOW>/ file theme.mp3
19:52:20.183 T:9840   DEBUG: PLEX.kodi_db.video: Did not find any file, abort
19:52:20.183 T:9840   DEBUG: PLEX.kodi_db: No kodi video db element found for path /media/<SOME_TV_SHOW>/ file theme.mp3
19:52:20.185 T:9840   DEBUG: PLEX.kodi_db.music: Found wrong number of path ids: [] for path /media/<SOME_TV_SHOW>/, abort
19:52:20.186 T:9840   DEBUG: PLEX.playlist_func: Research results for kodi_item: {u'title': u'', u'type': None, u'id': None, u'file': u'/media/<SOME_TV_SHOW>/theme.mp3', u'label': u'/media/<SOME_TV_SHOW>/theme.mp3'}
19:52:20.186 T:9840   DEBUG: PLEX.playlist_func: Made playlist item from Kodi: {'id': None, 'plex_id': None, 'plex_type': 'None', 'plex_uuid': 'None', 'kodi_id': None, 'kodi_type': 'None', 'file': '/media/<SOME_TV_SHOW>/theme.mp3', 'uri': 'library://whatever/item/%2Fmedia%2F%3CSOME_TV_SHOW%3E%2Ftheme.mp3', 'guid': 'None', 'playmethod': 'None', 'playcount': None, 'offset': None, 'force_transcode': False, 'part': 0, 'xml': "None"}
19:52:20.191 T:9840   DEBUG: PLEX.playqueue: Done comparing playqueues
```
Kodi is set to ignore `theme.mp3` files in `advancedsettings.xml` and Plex seems to ignore them on TV Show libraries by default.

This feels more of a nuisance than an issue, because of the added noise in `kodi.log`, but enough to feel it's worth mentioning since I got those lines repeated 22111 times in a single 94 minutes Kodi session, bloating the log to hefty 46.7MB

**Comment by croneter** [comment]
Share your entire debug log please as detailed in https://github.com/croneter/PlexKodiConnect/blob/master/.github/ISSUE_TEMPLATE/bug_report.md - which you ignored

**Comment by BrutuZ** [comment]
The template wasn't displayed when opening the issue, only the blank fields. Can't really ignore what can't be seen ;) 

In any case, here's a fresh full [kodi.log](https://github.com/croneter/PlexKodiConnect/files/2881009/kodi.log) as requested. 

Opened Kodi, navigate to TV Shows library, let cursor over a show while its 30s `theme.mp3` plays in the background, move over 2 other shows, after the third theme finished playing just went back to main menu and let it sit for around 15 minutes to let the log build up.

**Comment by croneter** [comment]
@BrutuZ 
The debug messages don't appear in the log you linked. Please reproduce the issue and then upload the log, making sure you actually caught these messages.

**Comment by BrutuZ** [comment]
It's a 6Mb log for a 20min session with no actual playback or interaction other than the required steps to reproduce.
There are `3216` instances for both the beginning and ending lines of the loop (`Comparing new Kodi playqueue` and `Done comparing playqueues`) starting at line `2129`. Line `2523` has a couple of the loops in a row, without any message other than PKC's in the middle.

`3216` playqueue checks for just `3` different files played once for their 30 seconds duration.

[Same log with only PLEX related lines](https://gist.github.com/BrutuZ/4f221b72b0b71d0df1e17033d182b906)

**Comment by croneter** [comment]
Should be solved with stable and beta version 2.7.3. Let me know here if this is not the case
